hr_timesheet_holiday/tests/test_all.py

```diff
@@ -3,7 +3,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 import time
 from openerp.tests.common import TransactionCase
-from openerp.exceptions import UserError, ValidationError
+from openerp.exceptions import Warning as UserError, ValidationError
```

Comment:
Seems like a voodoo magic, why not to change all UserError to Warning?

Reply:
[OCA guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#33idioms) point 7: Exceptions.

Reply:
@alexey-pelykh that's a wrong choice of class name in earlier versions. You missed all the fun back then ;-)

Reply:
(the reason it was a wrong choice is that it masks https://docs.python.org/2/library/exceptions.html#exceptions.Warning)

Reply:
Yep, missed all the mysterious fun :) 👍